src/Symfony/Component/ObjectMapper/Metadata/Mapping.php

```diff
@@ -11,26 +11,15 @@
 
 namespace Symfony\Component\ObjectMapper\Metadata;
 
+use Symfony\Component\ObjectMapper\Attribute\Map;
+
 /**
  * Configures a class or a property to map to.
  *
  * @internal
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
  */
-readonly class Mapping
+final class Mapping extends Map
```

Comment:
Do we really want to introduce this type inheritance?

Reply:
the class is internal so we can undo it at will. For now, it prevents duplicate definitions/docblocks IMHO